website/templates/include/profile/schools.mako

```diff
@@ -96,7 +95,7 @@
             </div>
 
             <div>
-                <a class="btn btn-default" data-bind="click: addContent">
+                <a class="btn btn-default" data-bind="css: enablebtn, click: addContent">
```

Comment:
Instead of applying a class conditionally, could you use the `disable` binding?

ref: http://knockoutjs.com/documentation/disable-binding.html